samples/CMakeLists.txt

```diff
@@ -5,6 +5,20 @@
 #
 
 if (CONFIG_SPM)
+  get_cmake_property(VARIABLES              VARIABLES)
+  get_cmake_property(VARIABLES_CACHED CACHE_VARIABLES)
+
+  set(regex "^CONFIG_SPM_SERVICE.+")
+
+  list(FILTER VARIABLES        INCLUDE REGEX ${regex})
+  list(FILTER VARIABLES_CACHED INCLUDE REGEX ${regex})
```

Comment:
This is ok for now, but `VARIABLES` is a very long list.
Is this really the only way to do this ?

Reply:
It's the only way I could think of where we won't have to remember to edit this code every time a new secure service is added. I totally agree it's not ideal.

Reply:
yep, let's look into an improvement later.
Could you create an issue and assign to me ?

Reply:
Done